pages/linux/snapper.md

```diff
@@ -0,0 +1,27 @@
+# snapper
+
+> Filesystem snapshot management tool.
+
+- List snapshot configs:
+
+`snapper list-configs`
+
+- Create snapper config:
+
+`snapper -c {{config}} create-config {{path/to/subvolume}}`
```

Comment:
Let's mention a concrete example of path/to/subvolume

Reply:
Thanks for the comments - I'm not a real world example would work to well, as the subvolume could be any directory in the filesystem.
As example being `snapper -c home create-config /home`

Reply:
Is it always a directory?

Reply:
Yes - a subvolume would always be a directory

Reply:
Then how about just `path/to/directory` ?

Reply:
I have added and pushed this :)